\section*{Discussion}

In this investigation of chemotherapy-associated mutations and resulting neoantigens in ovarian cancer, we find that the number of neoantigens after surgery, chemotherapy, and relapse is as much as double that of the primary tumor. However, the number of mutations matching signatures observed in \textit{G. gallus} and \textit{C. Elegans} exposured to cisplatin or cyclophosphamide usually account for less than 5\% of the overall mutation and neoantigen burden after treatment. In the most heavily treated patient, chemotherapy-associated SNVs accounted for 22\% of mutations and 15\% of neoantigens.

A surprising finding was that in samples of post-chemotherapy ascitic fluid, the number of neoantigens with evidence for RNA expression tracks the increase in DNA neoantigens, but decreases in solid tumors, despite the overall increase in neoantigens in the DNA. This raises the possibility of immunoediting\cite{Dunn_2002} at the level of transcription occurring post-chemotherapy in solid tumors. This idea is consistent with studies showing that paclitaxel/carboplatin and other chemotherapies may enhance T cell infiltration and cytotixicity in solid tumors\cite{Demaria2001,Wu_2009,Pfannenstiel_2010,Hodge_2013}, whereas ascites remain immune-suppressed\cite{Giuntoli2009,Simpson-Abelson2013,Singel2016}. However, this result is based on only four post-treatment solid tumor samples with RNA sequencing, and the analysis method is sensitive to changes in the expression of unrelated genes. Larger cohorts with paired pre- and post-chemotherapy RNA sequencing of solid tumors are needed to determine if transcriptional downregulation of neoantigens after treatment is a recurring phenomenon.

This study makes several technical contributions, which we hope may be a foundation for future work. We proposed a hierarchical Bayesian regression model to combine paired count data while controlling for confounding variables. We developed a method for extracting mutational signatures suitable for use with deconstructSigs from treated/control mutagenicity studies. The nine extracted signatures are made available with this publication. Our neoantigen identification pipeline is also made available, which includes support for indels and the identification of neoantigens with RNA support. 

There are several limitations to the analyses in this study. The signature extraction used only SNVs; MNVs and indels were considered separately. We did not consider large genomic rearrangements, which are known to result in neoantigens \cite{Worley2001}. The Bayesian model controlled for 

There are several limitations... RNA with hard thresholds to detect expressed peptides is subject to confounding by changes in gene expression -- e.g. if something gets upregulated post treatment. Also not including SVs and fusions, which may be immongenic, and likely under-calling indels. Substantial fraction of mutations are unresolved by signature deconvolution, and this amount increases with treatment

TODO: some sort of comparison of our neoantigen counts against another study in e.g. TCGA, and perhaps a calculation for how much more likely someone is to benefit from chekcpoint blockade based on these mutations


