\section*{Methods}
All in-house developed tools are available at https://github.com/hammerlab.

\subsection*{Data acquisition and mutation calling}
We used the original mutation calls generated by \cite{Patch_2015}. DNA and RNA sequencing reads were downloaded from the European Genome-phenome Archive (accession EGAD0000100087). We considered a mutation to be present in a sample if it was called in any sample from the same donor and if any DNA or RNA reads from the sample contained the variant. The RNA BAMs were realigned to the Grch37 reference using the STAR aligner.

\subsection*{HLA typing and MHC binding prediction}
HLA typing was performed using seq2HLA\cite{Boegel_2012} on tumor RNA-seq, variant annotation was performed using Varcode (https://github.com/hammerlab/varcode), and Class I MHC binding predictions were done with NetMHC 3.4\cite{Lundegaard_2008}.

\subsection*{Mutational signature deconvolution}
Signatures were deconvolved using the deconstructSigs\cite{Rosenthal_2016} package using the 30 signatures curated by COSMIC (http://cancer.sanger.ac.uk/cosmic/signatures) plus a signature manually extracted from a study of cisplatin exposure in \textit{C. Elegans}\cite{Meier_2014}. The manual extraction of this signature was done by counting the number of occurrences of each of the 96 possible mutations in trinucleotide contexts in the normal genomic background (Genotype=”N2”) and treated by Cisplatin at any concentration and normalizing by the trinucleotide content of the \texit{C. Elegans} genome. We did not consider mutations introduced in Cisplatin-treated individuals of other genomic backgrounds.

