\section*{Methods}
All in-house developed tools are available at https://github.com/hammerlab. Data and analysis notebooks are available at https://github.com/hammerlab/ovarian-paired-analyses.

\subsection*{Next-generation sequencing}
We used the original mutation calls generated by \cite{Patch_2015}. DNA and RNA sequencing reads were downloaded from the European Genome-phenome Archive (accession EGAD0000100087). We considered a mutation to be present in a sample if it was called in any sample from the same donor and if any DNA or RNA reads from the sample contained the variant.

\subsection*{Bayesian modeling}
If one makes the assumption that adjuvant and neoadjuvant treatments have identical effects on mutational burden, then in principle the five neoadjuvant-treated primary samples enable a disentangling of the effects of treatment and relapse time-point, but such a model had large uncertainties and proved inconclusive (Figure~\ref{fig:bayesian_model_effects_separate}). 

This model integrated information from paired pre- and post-treatment samples and unpaired samples, controlling for sample tissue type (solid or ascites), sample purity, the number of samples from each donor, and allowing for an interaction (effect modification) between tissue type and 
 
\subsection*{HLA typing and MHC binding prediction}
HLA typing was performed using a consensus of seq2HLA\cite{Boegel_2012} and OptiType\cite{Szolek_2014} across the available samples for each donor. Variant annotation was performed using Varcode (https://github.com/hammerlab/varcode). Class I MHC binding predictions were done with NetMHCpan 2.8\cite{Lundegaard_2008}.

\subsection*{Signature extraction}
Both ~\cite{Meier_2014} and ~\cite{Szikriszt_2016} performed sequencing on several replicates of chemotherapy-treated and untreated organisms. The signature extraction task is to split the mutations observed in the treated group into background and treatment-specific effects.

We consider each study and chemotherapy drug separately. Let $C_{i,j}$ be the number of mutations observed in experiment $0 \leq i \lt E$ for mutational trinucletoide context $0 \leq j \lt 96$. Let $s_i \in \{0,1\}$ be 1 if the treatment was administered in experiment $i$ and 0 if it was a control. We estimate the number of mutations in each context typically arising due to background (non-treatment) processes $B_j$ and the number due to treatment $T_j$ according to the model:

\[
C_{i,j} \sim \mathit{Poisson}(B_j + s_i T_j)
\]

We fit this model using Stan\cite{Gelman_2015}.

Let $h_j$ be the number of times the reference trinucleotide $j$ occurs in the human genome, and let $m_j$ be the number of times it occurs in the animal model genome. The human mutational signature for treatment $N_j$ is calculated as:

\[
N_j = \frac{T_j}{\sum_j{T_j}} \cdot \frac{h_j}{a_j}
\]


\subsection*{Signature deconvolution}
Signatures were deconvolved using the deconstructSigs\cite{Rosenthal_2016}

\subsection*{Counting neoantigens attributable to mutational signatures}
The number of SNVs and neoantigens generated by each signature was estimated by summing over the mutations or neoantigens in a sample the posterior probability that the signature generated the mutation.

Let $D_{i,s}$ be the signature deconvolution matrix giving the weight for signature $s$ in sample $i$. Let $H_{s,j}$ be the contribution of mutational context $j$ to signature $s$. Suppose mutation $m$ occurs in context $c_m$ and sample $l_m$. To estimate the probability that mutation $m$ was generated by signature $s$ we apply Bayes' Rule:

\[

\Pr[\text{$m$ due to signature $s$} | D,H ] = 

\]


MENTION the control



