\section*{Methods}
All in-house developed tools are available at https://github.com/hammerlab. Data and analysis notebooks are available at https://github.com/hammerlab/ovarian-paired-analyses.

\subsection*{Next-generation sequencing}
We used the original mutation calls generated by \cite{Patch_2015}. DNA and RNA sequencing reads were downloaded from the European Genome-phenome Archive (accession EGAD0000100087). We considered a mutation to be present in a sample if it was called in any sample from the same donor and if there were more than 10 DNA reads supporting the alternate allele.

\subsection*{Bayesian modeling}
Our model design is driven by two aspects of the AOCS dataset. First, 76/80 pre-treatment samples are solid tumor, whereas 24/35 post-treatment samples are drained ascitic fluid. This imbalance occurs because surgery is rarely performed for relapsed disease. We find evidence that ascites samples tend to harbor more detectable mutations than solid tumors, a phenomenon Patch et al suggest may be due to increased mixing of subclones in ascites. This must be accounted for if one is to compare the mostly solid tumor pre-treatment samples to predominantly ascitic post-treatment samples. Our model therefore controls for tissue type and other variables when analyzing changes in mutations and neoantigens with treatment. Second, 12/93 donors in the cohort have both pre-treatment and post-treatment samples; the remaining donors have only treated or only untreated samples. In assessing changes in mutations after treatment, we would like to maximize statistical power with a paired (within-subject) analysis when possible while also making use of the unpaired samples. The model we propose therefore includes a hierarchical component that models donor-specific effects.

If one makes the assumption that adjuvant and neoadjuvant treatments have identical effects on mutational burden, then in principle the five neoadjuvant-treated primary samples enable a disentangling of the effects of treatment and relapse time-point, but such a model had large uncertainties and proved inconclusive (Figure~\ref{fig:bayesian_model_effects_separate}). 

This model integrated information from paired pre- and post-treatment samples and unpaired samples, controlling for sample tissue type (solid or ascites), sample purity, the number of samples from each donor, and allowing for an interaction (effect modification) between tissue type and 
 
\subsection*{HLA typing and MHC binding prediction}
HLA typing was performed using a consensus of seq2HLA\cite{Boegel_2012} and OptiType\cite{Szolek_2014} across the available samples for each donor. Variant annotation was performed using Varcode (https://github.com/hammerlab/varcode). Class I MHC binding predictions were done with NetMHCpan 2.8\cite{Lundegaard_2008}.

\subsection*{Mutational signatures}
Most previous studies have done de-novo signature extraction then compared to existing signatures. We instead deconvolve directly onto existing signatures.

Both ~\cite{Meier_2014} and ~\cite{Szikriszt_2016} sequenced replicates of chemotherapy-treated and untreated (control) organisms. To identify a mutational signature associated with treatment, we must split the mutations observed in the treated group into background (i.e. those we would expect to see without treatment) and treatment effects. We do this for each study and chemotherapy drug separately.

Let $C_{i,j}$ be the number of mutations observed in experiment $0 \leq i \lt E$ for mutational trinucletoide context $0 \leq j \lt 96$. Let $s_i \in \{0,1\}$ be 1 if the treatment was administered in experiment $i$ and 0 if it was a control. We estimate the number of mutations in each context typically arising due to background (non-treatment) processes $B_j$ and the number due to treatment $T_j$ according to the model:

\[
C_{i,j} \sim \mathit{Poisson}(B_j + s_i T_j)
\]

We fit this model using Stan\cite{Gelman_2015} with a uniform (improper) prior on the entries of $B$ and $T$. The mutational signature of treatment $N$ was calculated from a point estimate of $T$ as:

\[
N_j = \left ( \frac{T_j}{\sum_{j'}{T_{j'}}} \right ) \left ( \frac{h_j}{a_j} \right )
\]

where $h_j$ and $m_j$ are the number of times the reference trinucleotide $j$ occurs in the human and animal (\textit{C. Elegans} or \textit{G. Gallus}) genomes, respectively.

The signature deconvolution itself was performed with the deconstructSigs\cite{Rosenthal_2016} package using the following parameters passed to \texttt{whichSignatures()}: \texttt{contexts.needed=TRUE}, \texttt{signature.cutoff=0.05}, \texttt{tri.counts.method = "default"}.

\subsection*{Counting neoantigens attributable to mutational signatures}
To estimate the number of SNVs generated by a signature, we summed the posterior probability that the signature generated each mutation over the mutations in the sample. For neoantigens, we weighted the sum by the number of neoantigens resulting from each SNV.

Let $H_{s,j}$ be the input signatures, giving weights for each signature $s$ on mutational context $j$. Let $D_{i,s}$ be the signature deconvolution result, giving the contribution of signature $s$ to sample $i$. Suppose a mutation occurs in context $c$ and sample $l$. Let $\Pr[s \mid c]$ be the probability that signature $s$ gave rise to this mutation. We calculate this using Bayes' Rule:

\[
\Pr[s \mid c] = \frac{\Pr[c \mid s] \Pr[s]}{\sum_{s'}{\Pr[c \mid s']\Pr[s']}} = \frac{H_{s,c} D_{l,s}}{\sum_{s'}{H_{s',c} D_{l,s'}}}
\]
