\section*{Methods}

\subsection*{Next-generation sequencing}
We used the original mutation calls generated by \cite{Patch_2015}. DNA and RNA sequencing reads were downloaded from the European Genome-phenome Archive (accession EGAD0000100087). We considered a mutation to be present in a sample if it was called in any sample from the same donor and if there were more than 10 DNA reads supporting the alternate allele.

\subsection*{Bayesian model of the change in mutations, neoantigens, and expressed neoantigens after treatment}
Figure~\ref{fig:model_architecture} shows the model architecture and figure ~\ref{fig:bayesian_model_effects} the results. Two aspects of the AOCS dataset motivated this model. First, 76/80 pre-treatment samples are solid tumor, whereas 24/35 post-treatment samples are drained ascitic fluid. This imbalance occurs because surgery is rarely performed for relapsed disease. We find evidence that ascites samples tend to harbor more detectable mutations than solid tumors, a phenomenon Patch et al suggest may be due to increased mixing of subclones in ascites. This must be accounted for if one is to compare the mostly solid tumor pre-treatment samples to predominantly ascitic post-treatment samples. Our model therefore controls for tissue type and other variables when analyzing changes in mutations and neoantigens with treatment. Second, 12/93 donors in the cohort have both pre-treatment and post-treatment samples; the remaining donors have only treated or only untreated samples. In assessing changes in mutations after treatment, we would like to maximize statistical power with a paired (within-subject) analysis when possible while also making use of the unpaired samples. The model we propose therefore includes a hierarchical component that models donor-specific effects. Several posterior predictive checks were performed, which indicated the model appears reasonable for this dataset. 

We note that if one makes the assumption that adjuvant and neoadjuvant treatments have identical effects on mutational burden, then in principle the five neoadjuvant-treated primary samples might enable disentangling the effects of treatment and relapse time-point. However, such a model had large uncertainties and proved inconclusive (Figure~\ref{fig:bayesian_model_effects_separate}). 

\subsection*{HLA typing, variant annotation, and MHC binding prediction}
HLA typing was performed using a consensus of seq2HLA\cite{Boegel_2012} and OptiType\cite{Szolek_2014} across the available samples for each donor. Variant annotation was performed using Varcode (https://github.com/hammerlab/varcode). For indels, all peptides potentially generated up to a stop codon were considered. Class I MHC binding predictions were done with NetMHCpan 2.8\cite{Lundegaard_2008}.

\subsection*{Mutational signatures}
Previous studies have often performed signature extraction \textit{de novo}, and subsequently compared the extracted signatures to existing signatures. We instead deconvolve onto existing signatures. This is more appropriate for relatively small datasets such as ours and enables a straightforward interpretation of the deconvolution results.

Both Meier et al~\cite{Meier_2014} and Szikriszt et al~\cite{Szikriszt_2016} sequenced replicates of chemotherapy-treated and untreated (control) organisms. Identifying a mutational signature associated with treatment requires splitting the mutations observed in the treated group into background (i.e. those we would expect to see without treatment) and treatment effects. We do this for each study and chemotherapy drug separately.

Let $C_{i,j}$ be the number of mutations observed in experiment $i$ for mutational trinucletoide context $0 \leq j < 96$. Let $t_i \in \{0,1\}$ be 1 if the treatment was administered in experiment $i$ and 0 if it was a control. We estimate the number of mutations in each context typically arising due to background (non-treatment) processes $B_j$ and the number due to treatment $T_j$ according to the model:

\[
C_{i,j} \sim \mathit{Poisson}(B_j + t_i T_j)
\]

We fit this model using Stan\cite{Gelman_2015} with a uniform (improper) prior on the entries of $B$ and $T$. The mutational signature of treatment $N$ was calculated from a point estimate of $T$ as:

\[
N_j = \left ( \frac{T_j}{\sum_{j'}{T_{j'}}} \right ) \left ( \frac{h_j}{a_j} \right )
\]

where $h_j$ and $m_j$ are the number of times the reference trinucleotide $j$ occurs in the human and animal (\textit{C. Elegans} or \textit{G. Gallus}) genomes, respectively.

The signature deconvolution itself was performed with the deconstructSigs\cite{Rosenthal_2016} package using the following parameters passed to \texttt{whichSignatures()}: \texttt{contexts.needed=TRUE}, \texttt{signature.cutoff=0.05}, \texttt{tri.counts.method="default"}.

The deconstructSigs package uses a forward-selection procedure with stopping criteria to select the signatures that best explain the mutations in a sample, followed by discarding signatures with contribution less than a threshold. The solutions returned by deconstructSigs typically do not account for all mutations in a sample, i.e. the weights for each signature sum to less than 1. We refer to the mutations not accounted for by the deconvolution as the ``residual'' mutations. For each sample, we additionally extracted a ``residual signature'' $R_i$, enabling us to quantify the number of mutations or neoantigens resulting from these mutations. 

\[
R_i = \min(0, C_{i, \cdot} - \sum_s{D_{i,s} H_{s, \cdot}} 
\]

where $H_{s,j}$ is the weight for signature $s$ on mutational context $j$, $D_{i,s}$ gives the contribution of signature $s$ to sample $i$, and the minimum is taken elementwise across the vector.

To estimate the number of SNVs and neoantigens generated by a signature, we calculated the posterior probability that the signature generated each mutation over the mutations in the sample, as described below. The sum of these probabilities gives the expected number of SNVs attributable to each signature. For neoantigens, we weight the terms of this sum by the number of neoantigens generated by each mutation.

Suppose a mutation occurs in context $c$ and sample $l$. Let $\Pr[s \mid c]$ be the probability that signature $s$ gave rise to this mutation. We calculate this using Bayes' Rule:

\[
\Pr[s \mid c] = \frac{\Pr[c \mid s] \Pr[s]}{\sum_{s'}{\Pr[c \mid s']\Pr[s']}} = \frac{H_{s,c} D_{l,s}}{\sum_{s'}{H_{s',c} D_{l,s'}}}
\]
