\section*{Methods}

\subsection*{Next-generation sequencing}
We used the original mutation calls generated by Patch et al~\cite{Patch_2015}. DNA and RNA sequencing reads were downloaded from the European Genome-phenome Archive (accession EGAD0000100087). We considered a mutation to be present in a sample if it was called in any sample from the same patient and if there were more than 10 DNA reads supporting the alternate allele. We considered a mutation to be expressed if there were 3 or more RNA reads supporting the variant allele.

\subsection*{Bayesian model of the change in mutations, neoantigens, and expressed neoantigens after treatment}
Two aspects of the AOCS dataset motivated a hierarchical Bayesian model to estimate the overall change in mutations and neoantigens after chemotherapy treatment. First, 76/80 pre-treatment samples are solid tumor, whereas 24/35 post-treatment samples are drained ascitic fluid. This imbalance occurs because surgery is rarely performed for relapsed disease. We find evidence that ascites samples tend to harbor more detectable mutations than solid tumors, a phenomenon Patch et al suggest may be due to increased mixing of subclones in ascites. This must be accounted for if one is to compare the mostly solid tumor pre-treatment samples to predominantly ascitic post-treatment samples. Our model therefore controls for tissue type and other variables when analyzing changes in mutations and neoantigens with treatment. Second, 12/93 patients in the cohort have both pre-treatment and post-treatment samples; the remaining patients have only treated or only untreated samples. In assessing changes in mutations after treatment, we would like to maximize statistical power with a paired (within-subject) analysis when possible while also making use of the unpaired samples. The model we propose therefore includes a hierarchical component that models patient-specific effects. Several posterior predictive checks were performed, which indicated the model appears reasonable for this dataset. Figure~\ref{fig:model_architecture} shows the model architecture and figure~\ref{fig:bayesian_model_effects} the results. 

% We note that if one makes the assumption that adjuvant and neoadjuvant treatments have identical effects on mutational burden, then in principle the five neoadjuvant-treated primary samples might enable disentangling the effects of treatment and relapse time-point. However, such a model had large uncertainties and proved inconclusive (Figure~\ref{fig:bayesian_model_effects_separate}). 

\subsection*{HLA typing, variant annotation, and MHC binding prediction}
HLA typing was performed using a consensus of seq2HLA\cite{Boegel_2012} and OptiType\cite{Szolek_2014} across the available samples for each patient. The most disruptive effect of each variant was predicted using Varcode (https://github.com/hammerlab/varcode). For indels, all peptides potentially generated up to a stop codon were considered. Class I MHC binding predictions were performed for peptides of length 8--11 using NetMHCpan 2.8\cite{Lundegaard_2008} with default arguments.

\subsection*{Mutational signatures}
Previous studies have mostly performed signature extraction \textit{de novo} and interpreted the results by comparing to existing signatures. We instead deconvolve onto existing signatures. This is more appropriate for our relatively small dataset and enables a straightforward interpretation.

Both Meier et al~\cite{Meier_2014} and Szikriszt et al~\cite{Szikriszt_2016} sequenced replicates of chemotherapy-treated and untreated (control) organisms. Identifying a mutational signature associated with treatment requires splitting the mutations observed in the treated group into background (i.e. those we would expect to see without treatment) and treatment effects. We do this for each study and chemotherapy drug separately (Supplemental Methods).

The signature deconvolution was performed with the deconstructSigs\cite{Rosenthal_2016} package.
