There is increasing appreciation for the importance of T cell neoantigens arising from somatic mutations in anti-tumor immune responses~\cite{Schumacher_2015}. Patients with more neoantigens have better prognosis~\cite{Brown_2014} and are more likely to respond to checkpoint blockade immunotherapy\cite{Van_Allen_2015,Rizvi_2015}. Several chemotherapies including platinum compounds have long been known to be mutagenic\cite{Hannan_1989} and recent studies have found evidence for chemotherapy-induced mutations in human cancers\cite{Murugaesu_2015,Johnson_2013}. While there has been much study on how chemotherapy may intrinsically enhance~\cite{Hato_2012,Machiels2001,Hodge2013} or impair~\cite{Litterman_2013} an anti-tumor immune response, the extent to which it induces tumor neoantigens has not been well assessed.

The Australian Ovarian Cancer Study (AOCS) performed whole genome sequencing on 115 cancer samples from 93 donors with high grade serous ovarian carcinoma\cite{Patch_2015}. Eighty samples were taken before chemotherapy and 35 after combination carboplatin/paclitaxel and other chemotherapy treatments. The primary analysis found that post-treatment samples harbored more somatic mutations than pre-treatment samples and exhibited evidence of chemotherapy-associated mutations. Here we extend these results by quantifying the change in mutational and neoantigen burden with treatment and estimating the number of neoantigens attributable to chemotherapy-associated mutational signatures.

As paclitaxel/carboplatin therapy following surgery is standard of care, all AOCS relapse samples were exposed to chemotherapy. Therefore, the effect of chemotherapy on mutational burden cannot be statistically separated from those of drift and relapse in this cohort. Mutational signature deconvolution provides a way to estimate the number of mutations likely due to chemotherapy. In this method, the SNVs observed in a sample are expressed as a combination of \textit{signatures}, each corresponding to a mutagenic process~\cite{Alexandrov2013}. The signatures are defined by their preference for mutations at certain tri-nucleotide contexts. Mutational signatures may be discovered \textit{de novo} from large cancer sequencing projects, but for smaller studies it is preferable to apply deconvolution using known signatures~\cite{Rosenthal_2016}, which we do here. The COSMIC Signature Resource provides thirty signatures from large pan-cancer analyses; a number of these have known associations with mutagenic processes such as BRCA disruption or ultraviolet light exposure~\cite{364242}. While signatures for chemotherapy exposure have not been established from human studies, two recent reports provide data on mutations detected in chemotherapy-exposed \textit{C. Elegans} organisms~\cite{Meier_2014} and a \textit{G. Gallus} (chicken) cell line~\cite{Szikriszt_2016}. The \textit{C. Elegans} study considered cisplatin and other compounds across a range of DNA repair-deficient knockout models. The \textit{G. Gallus} study looked at a number of chemotherapies, including cisplatin, cyclophosphamide, and etoposide, in wildtype chicken cell lines. Using the mutations identified in these studies, we extracted signatures for cisplatin, cyclophosphamide, and etoposide, and applied these to the AOCS dataset.