%% BioMed_Central_Tex_Template_v1.06
%%                                      %
%  bmc_article.tex            ver: 1.06 %
%                                       %

%%IMPORTANT: do not delete the first line of this template
%%It must be present to enable the BMC Submission system to
%%recognise this template!!

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                     %%
%%  LaTeX template for BioMed Central  %%
%%     journal article submissions     %%
%%                                     %%
%%          <8 June 2012>              %%
%%                                     %%
%%                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                                                 %%
%% For instructions on how to fill out this Tex template           %%
%% document please refer to Readme.html and the instructions for   %%
%% authors page on the biomed central website                      %%
%% http://www.biomedcentral.com/info/authors/                      %%
%%                                                                 %%
%% Please do not use \input{...} to include other tex files.       %%
%% Submit your LaTeX manuscript as one .tex document.              %%
%%                                                                 %%
%% All additional figures and files should be attached             %%
%% separately and not embedded in the \TeX\ document itself.       %%
%%                                                                 %%
%% BioMed Central currently use the MikTex distribution of         %%
%% TeX for Windows) of TeX and LaTeX.  This is available from      %%
%% http://www.miktex.org                                           %%
%%                                                                 %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% additional documentclass options:
%  [doublespacing]
%  [linenumbers]   - put the line numbers on margins

%%% loading packages, author definitions

%\documentclass[twocolumn]{bmcart}% uncomment this for twocolumn layout and comment line below
\documentclass{bmcart}

%%% Load packages
%\usepackage{amsthm,amsmath}
%\RequirePackage{natbib}
%\RequirePackage[authoryear]{natbib}% uncomment this for author-year bibliography
%\RequirePackage{hyperref}
\usepackage[utf8]{inputenc} %unicode support
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                             %%
%%  If you wish to display your graphics for   %%
%%  your own use using includegraphic or       %%
%%  includegraphics, then comment out the      %%
%%  following two lines of code.               %%
%%  NB: These line *must* be included when     %%
%%  submitting to BMC.                         %%
%%  All figure files must be submitted as      %%
%%  separate graphics through the BMC          %%
%%  submission process, not included in the    %%
%%  submitted article.                         %%
%%                                             %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\def\includegraphic{}
\def\includegraphics{}



%%% Put your definitions there:
\startlocaldefs
\endlocaldefs


%%% Begin ...
\begin{document}

%%% Start of article front matter
\begin{frontmatter}

\begin{fmbox}
\dochead{Research}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the title of your article here     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\title{Chemotherapy weakly contributes to predicted neoantigen expression in ovarian cancer}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors here                   %%
%%                                          %%
%% Specify information, if available,       %%
%% in the form:                             %%
%%   <key>={<id1>,<id2>}                    %%
%%   <key>=                                 %%
%% Comment or delete the keys which are     %%
%% not used. Repeat \author command as much %%
%% as required.                             %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\author[addressref={aff1}, email={tim@hammerlab.org}]{\inits{TO}\fnm{Timothy} \snm{O'Donnell}}
\author[addressref={aff2}, email={elizabeth.christie@petermac.org}]{\inits{EC}\fnm{Elizabeth L.} \snm{Christie}}
\author[addressref={aff1}, email={aahuja11@gmail.com}]{\inits{AA}\fnm{Arun} \snm{Ahuja}}
\author[addressref={aff1}, email={jacki@hammerlab.org}]{\inits{JB}\fnm{Jacqueline} \snm{Buros}}
\author[addressref={aff1}, email={arman@hammerlab.org}]{\inits{BAA}\fnm{B. Arman} \snm{Aksoy}}
\author[addressref={aff2}, email={david.bowtell@petermac.org}]{\inits{DB}\fnm{David D. L.} \snm{Bowtell}}
\author[addressref={aff3}, noteref={n1}, email={snyderca@mskcc.org}]{\inits{AS}\fnm{Alexandra} \snm{Snyder}}
\author[addressref={aff1}, noteref={n1}, email={hammer@hammerlab.org}]{\inits{JH}\fnm{Jeff} \snm{Hammerbacher}}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors' addresses here        %%
%%                                          %%
%% Repeat \address commands as much as      %%
%% required.                                %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\address[id=aff1]{%                           % unique id
  \orgname{Icahn School of Medicine at Mount Sinai}, % university, etc
  % \street{1 Gustave L. Levy Pl},                     %
  %\postcode{}                                % post or zip code
  \city{New York},                              % city
  \cny{USA}                                    % country
}
\address[id=aff2]{%                           % unique id
  \orgname{Peter MacCallum Cancer Centre}, % university, etc
  % \street{1 Gustave L. Levy Pl},                     %
  \city{East Melbourne},                              % city
  \postcode{Victoria 3002}                                % post or zip code
  \cny{Australia}                                    % country
}
\address[id=aff3]{%                           % unique id
  \orgname{Department of Medicine, Memorial Sloan-Kettering Cancer Center, Weill Cornell Medical College}, % university, etc
  % \street{1 Gustave L. Levy Pl},                     %
  %\postcode{}                                % post or zip code
  \city{New York},                              % city
  \cny{USA}                                    % country
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter short notes here                   %%
%%                                          %%
%% Short notes will be after addresses      %%
%% on first page.                           %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{artnotes}
%\note{Sample of title note}     % note to the article
\note[id=n1]{Co-senior author} % note, connected to author
\end{artnotes}

\end{fmbox}% comment this for two column layout

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Abstract begins here                 %%
%%                                          %%
%% Please refer to the Instructions for     %%
%% authors on http://www.biomedcentral.com  %%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{abstractbox}
\begin{abstract} % abstract

\parttitle{Background}
Patients with highly mutated tumors, such as melanoma or smoking-related lung cancer, have higher rates of response to immune checkpoint blockade therapy, perhaps due to increased neoantigen expression. Many chemotherapies including platinum compounds are known to be mutagenic, but the impact of standard treatment protocols on mutational burden and resulting neoantigen expression in most human cancers is unknown.

\parttitle{Methods}
We sought to quantify the effect of chemotherapy treatment on computationally predicted neoantigen expression for 12 high grade serous ovarian carcinoma (HGSC) patients with pre- and post-chemotherapy samples collected in the Australian Ovarian Cancer Study. We additionally analyzed 16 patients from the cohort with post-treatment samples only, including five primary surgical samples exposed to neoadjuvant chemotherapy. Our approach integrates tumor whole genome and RNA sequencing with class I MHC binding prediction and mutational signatures of chemotherapy exposure extracted from two preclinical studies.

\parttitle{Results}
The mutational signatures for cisplatin and cyclophosphamide identified in a preclinical model had significant but inexact associations with the relevant exposure in the clinical samples. In an analysis stratified by tissue type (solid tumor or ascites), relapse samples collected after chemotherapy harbored a median of 90\% more expressed neoantigens than untreated primary samples, a figure that combines the effects of chemotherapy and other mutagenic processes operative during relapse. Neoadjuvant-treated primary samples showed no detectable increase over untreated samples. The contribution from chemotherapy-associated signatures was small, accounting for a mean of 5\% (range 0--16) of the expressed neoantigen burden in relapse samples. In both treated and untreated samples, most neoantigens were attributed to COSMIC \textit{Signature (3)}, associated with BRCA disruption, \textit{Signature (1)}, associated with a slow mutagenic process active in healthy tissue, and \textit{Signature (8)}, of unknown etiology.

\parttitle{Conclusion}
Relapsed HGSC tumors harbor nearly double the predicted expressed neoantigen burden of primary samples, but mutations associated with chemotherapy signatures account for only a small part of this increase. The mutagenic processes responsible for most neoantigens are similar between primary and relapse samples. Our analyses are based on mutations detectable from whole genome sequencing of bulk samples and do not account for neoantigens present in small populations of cells. 

\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The keywords begin here                  %%
%%                                          %%
%% Put each keyword in separate \kwd{}.     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{keyword}
\kwd{neoantigen}
\kwd{mutational signature}
\kwd{chemotherapy}
\end{keyword}

% MSC classifications codes, if any
%\begin{keyword}[class=AMS]
%\kwd[Primary ]{}
%\kwd{}
%\kwd[; secondary ]{}
%\end{keyword}

\end{abstractbox}
%
%\end{fmbox}% uncomment this for twcolumn layout

\end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The Main Body begins here                %%
%%                                          %%
%% Please refer to the instructions for     %%
%% authors on:                              %%
%% http://www.biomedcentral.com/info/authors%%
%% and include the section headings         %%
%% accordingly for your article type.       %%
%%                                          %%
%% See the Results and Discussion section   %%
%% for details on how to create sub-sections%%
%%                                          %%
%% use \cite{...} to cite references        %%
%%  \cite{koon} and                         %%
%%  \cite{oreg,khar,zvai,xjon,schn,pond}    %%
%%  \nocite{smith,marg,hunn,advi,koha,mouse}%%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% start of article main body
% <put your article body there>

%%%%%%%%%%%%%%%%
%% Background %%
%%
\section*{Background}

Many chemotherapies including platinum compounds~\cite{Hannan_1989}, cyclophosphamide~\cite{Anderson_1995}, and etoposide~\cite{NAKANOMYO_1986} exert their effect through DNA damage, and recent studies have found evidence for chemotherapy-induced mutations in post-treatment acute myeloid leukaemia~\cite{Ding_2012}, glioma~\cite{Johnson_2013}, and esophageal adenocarcinoma~\cite{Murugaesu_2015}. Successful development of immune checkpoint-mediated therapy\cite{Chen_2013} has focused attention on the importance of T cell responses to somatic mutations in coding genes that generate neoantigens~\cite{Schumacher_2015}. Studies based on bulk-sequencing of tumor samples followed by computational peptide-class I MHC affinity prediction~\cite{Lundegaard_2007} have suggested that tumors with more mutations and predicted mutant MHC I peptide ligands are more likely to respond to checkpoint blockade immunotherapy~\cite{Van_Allen_2015,Rizvi_2015}. Ovarian cancers fall into an intermediate group of solid tumors in terms of mutational load present in pre-treatment surgical samples\cite{Lawrence_2013}. However, the effect of standard chemotherapy regimes on tumor mutation burden and resulting neoantigen expression in ovarian cancer is poorly understood.

Investigators associated with the Australian Ovarian Cancer Study (AOCS) performed whole genome and RNA sequencing of 79 pre-treatment and 35 post-treatment cancer samples from 92 HGSC patients, including 12 patients with both pre- and post-treatment samples~\cite{Patch_2015}. The samples were obtained from solid tissue resections, autopsies, and ascites drained to relieve abdominal distension. Treatment regimes varied but primary treatment always included platinum-based chemotherapy. In their analysis, Patch et al. reported that post-treatment samples harbored more somatic mutations than pre-treatment samples and exhibited evidence of chemotherapy-associated mutations. Here we extend these results by quantifying the mutations and predicted neoantigens attributable to chemotherapy-associated mutational signatures. We find that, while neoantigen expression increases after treatment and relapse, only a small part of the increase is due to mutations associated with chemotherapy signatures.

\section*{Content}
Text and results for this section, as per the individual journal's instructions for authors. %\cite{koon,oreg,khar,zvai,xjon,schn,pond,smith,marg,hunn,advi,koha,mouse}

\section*{Section title}
Text for this section \ldots
\subsection*{Sub-heading for section}
Text for this sub-heading \ldots
\subsubsection*{Sub-sub heading for section}
Text for this sub-sub-heading \ldots
\paragraph*{Sub-sub-sub heading for section}
Text for this sub-sub-sub-heading \ldots
In this section we examine the growth rate of the mean of $Z_0$, $Z_1$ and $Z_2$. In
addition, we examine a common modeling assumption and note the
importance of considering the tails of the extinction time $T_x$ in
studies of escape dynamics.
We will first consider the expected resistant population at $vT_x$ for
some $v>0$, (and temporarily assume $\alpha=0$)
%
\[
 E \bigl[Z_1(vT_x) \bigr]= E
\biggl[\mu T_x\int_0^{v\wedge
1}Z_0(uT_x)
\exp \bigl(\lambda_1T_x(v-u) \bigr)\,du \biggr].
\]
%
If we assume that sensitive cells follow a deterministic decay
$Z_0(t)=xe^{\lambda_0 t}$ and approximate their extinction time as
$T_x\approx-\frac{1}{\lambda_0}\log x$, then we can heuristically
estimate the expected value as
%
\begin{eqnarray}\label{eqexpmuts}
E\bigl[Z_1(vT_x)\bigr] &=& \frac{\mu}{r}\log x
\int_0^{v\wedge1}x^{1-u}x^{({\lambda_1}/{r})(v-u)}\,du
\nonumber\\
&=& \frac{\mu}{r}x^{1-{\lambda_1}/{\lambda_0}v}\log x\int_0^{v\wedge
1}x^{-u(1+{\lambda_1}/{r})}\,du
\nonumber\\
&=& \frac{\mu}{\lambda_1-\lambda_0}x^{1+{\lambda_1}/{r}v} \biggl(1-\exp \biggl[-(v\wedge1) \biggl(1+
\frac{\lambda_1}{r}\biggr)\log x \biggr] \biggr).
\end{eqnarray}
%
Thus we observe that this expected value is finite for all $v>0$ (also see \cite{koon,khar,zvai,xjon,marg}).
%\nocite{oreg,schn,pond,smith,marg,hunn,advi,koha,mouse}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Backmatter begins here                   %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{backmatter}

\section*{Competing interests}
  The authors declare that they have no competing interests.

\section*{Author's contributions}
    Text for this section \ldots

\section*{Acknowledgements}
  Text for this section \ldots
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                  The Bibliography                       %%
%%                                                         %%
%%  Bmc_mathpys.bst  will be used to                       %%
%%  create a .BBL file for submission.                     %%
%%  After submission of the .TEX file,                     %%
%%  you will be prompted to submit your .BBL file.         %%
%%                                                         %%
%%                                                         %%
%%  Note that the displayed Bibliography will not          %%
%%  necessarily be rendered by Latex exactly as specified  %%
%%  in the online Instructions for Authors.                %%
%%                                                         %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% if your bibliography is in bibtex format, use those commands:
\bibliographystyle{bmc-mathphys} % Style BST file (bmc-mathphys, vancouver, spbasic).
\bibliography{bmc_article}      % Bibliography file (usually '*.bib' )
% for author-year bibliography (bmc-mathphys or spbasic)
% a) write to bib file (bmc-mathphys only)
% @settings{label, options="nameyear"}
% b) uncomment next line
%\nocite{label}

% or include bibliography directly:
% \begin{thebibliography}
% \bibitem{b1}
% \end{thebibliography}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Figures                       %%
%%                               %%
%% NB: this is for captions and  %%
%% Titles. All graphics must be  %%
%% submitted separately and NOT  %%
%% included in the Tex document  %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%
%% Do not use \listoffigures as most will included as separate files

\section*{Figures}
  \begin{figure}[h!]
  \caption{\csentence{Sample figure title.}
      A short description of the figure content
      should go here.}
      \end{figure}

\begin{figure}[h!]
  \caption{\csentence{Sample figure title.}
      Figure legend text.}
      \end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Tables                        %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% Use of \listoftables is discouraged.
%%
\section*{Tables}
\begin{table}[h!]
\caption{Sample table title. This is where the description of the table should go.}
      \begin{tabular}{cccc}
        \hline
           & B1  &B2   & B3\\ \hline
        A1 & 0.1 & 0.2 & 0.3\\
        A2 & ... & ..  & .\\
        A3 & ..  & .   & .\\ \hline
      \end{tabular}
\end{table}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Additional Files              %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Additional Files}
  \subsection*{Additional file 1 --- Sample additional file title}
    Additional file descriptions text (including details of how to
    view the file, if it is in a non-standard format or the file extension).  This might
    refer to a multi-page table or a figure.

  \subsection*{Additional file 2 --- Sample additional file title}
    Additional file descriptions text.


\end{backmatter}
\end{document}
